### DISCUSSION TITLE
Welcome to ExpressWebJs Discussions!

### DISCUSSION THREAD

**Discussion by Alexigbokwe** [Alexigbokwe/ExpressWebJs#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Alexigbokwe** [comment]
We are happy to announce the official release of Expresswebjs. This new major version of the framework provides a lot of features and improved performance. Visit the official website at [expresswebjs.com](expresswebjs.com) for full documentation and use.

**Reply by Tyav** [reply]
The url https://github.com/Alexigbokwe/ExpressWebJs/discussions/expresswebjs.com returns a not found

**Reply by Alexigbokwe** [reply]
> The url https://github.com/Alexigbokwe/ExpressWebJs/discussions/expresswebjs.com returns a not found

sorry about that, the link is https://github.com/Alexigbokwe/ExpressWebJs/discussions
